v1/common/amqp.go

```diff
@@ -4,23 +4,47 @@ import (
 	"crypto/tls"
 	"fmt"
 	"sync"
+	"time"
 
 	"github.com/streadway/amqp"
 )
 
 // AMQPConnector ...
 type AMQPConnector struct {
 	connManager *amqpConnectionManager
+
+	exchangeMaxRetries   int
+	exchangeRetryTimeout time.Duration
 }
 
 func NewAMQPConnector(url string, tlsConfig *tls.Config) *AMQPConnector {
 	return &AMQPConnector{
 		connManager: newAMQPConnectionManager(url, tlsConfig),
+
+		exchangeMaxRetries:   3,
+		exchangeRetryTimeout: time.Second,
```

Comment:
This one is not used

Reply:
the `time.Sleep` disappeared during a last-second refactoring, re-adding it